runtime/src/accounts_db.rs

```diff
@@ -1475,13 +1650,34 @@ impl AccountsDB {
                         .map(|i| i.accounts.capacity())
                         .sum::<u64>();
                 }
-                all_removed_slot_storages.push(slot_removed_storages);
+                all_removed_slot_storages.push(slot_removed_storages.clone());
             }
         }
         remove_storages_elapsed.stop();
 
         let num_slots_removed = all_removed_slot_storages.len();
 
+        let mut recycled_count = 0;
+        let mut recycle_stores = self.recycle_stores.write().unwrap();
```

Comment:
same as https://github.com/solana-labs/solana/pull/12885/files#r514094338, would be nice to add a timing field here as part of the `purge_slots_time` metric